src/Features/Core/Portable/Workspace/BackgroundParser.cs

```diff
@@ -208,14 +208,6 @@ private Task ParseDocumentAsync(Document document)
                 {
                     try
                     {
-                        if (SolutionCrawlerOptions.GetBackgroundAnalysisScope(document.Project) == BackgroundAnalysisScope.ActiveFile
-                            && _documentTrackingService?.TryGetActiveDocument() != document.Id)
-                        {
-                            // Active file analysis is enabled, but the document for parsing is not the current
-                            // document. Return immediately without parsing.
-                            return;
-                        }
```

Comment:
❔ I'm assuming this was removed because the compiler analyzer is going to need this even if no other analyzers do?

Reply:
Yes, that, plus also the fact that we are narrowing down the option to only affect background analyzer execution, not other kinds of background analysis in VS.